libs/service-portal/settings/src/components/UserOnboardingModal/UserOnboardingModal.tsx

```diff
@@ -52,6 +58,9 @@ const UserOnboardingModal: ServicePortalModuleComponent = ({ userInfo }) => {
 
   const closeModal = () => {
     setToggleCloseModal(true)
+    if (pathname) {
```

Comment:
This will only catch events where the modal is closed on the final step. Might  belong better in `dropOnboardingSideEffects`
See implementation of `<Modal />` and `<BaseModal />`